operations/iscsi_sbps/Node_Personalization_Verification.md

```diff
@@ -1,238 +1,4 @@
-        "limit": "",
-        "passthrough": "",
-        "verbosity": 0
-      },
-      "configuration": {
-        "limit": "",
-        "name": "iscsi-sbps-targets-config"
-      },
-      "debug_on_failure": false,
-      "logs": "ara.cmn.surtur.hpc.amslabs.hpecorp.net/?label=iscsi-sbps-targets-config",
-      "name": "iscsi-config-20240831210845",
-      "status": {
-        "artifacts": [],
-        "session": {
-          "completion_time": null,
-          "ims_job": null,
-          "job": null,
-          "start_time": "2024-08-31T21:08:47",
-          "status": "pending",
-          "succeeded": "none"
-        }
-      },
-      "tags": {},
-      "target": {
-        "definition": "dynamic",
-        "groups": [],
-        "image_map": []
-      }
-    }
-    ```
-
-Note:
-For specific target worker nodes selection for node personalization (instead of default: to all workers)
-use CFS option `--ansible-limit` with xnames of the worker nodes while creating a session.
-
-```bash
-cray cfs v3 sessions create --name <session_name> --configuration-name <config_name> --ansible-limit <xname1,xname2,...>
-```
-
-### Monitor CFS session
-
-#### Check status of CFS session
-
-See [View CFS Sessions](CFS_Sessions.md) for more information.
-
-```bash
-cray cfs v3 sessions describe "${SESSION}" --format toml
-```
-
-```toml
-debug_on_failure = false
-logs = "ara.cmn.surtur.hpc.amslabs.hpecorp.net/?label=iscsi-sbps-targets-config"
-name = "iscsi-config-20240831210845"
-
-[ansible]
-config = "cfs-default-ansible-cfg"
-limit = ""
-passthrough = ""
-verbosity = 0
-
-[configuration]
-limit = ""
-name = "iscsi-sbps-targets-config"
-
-[status]
-artifacts = []
-
-[tags]
-
-[target]
-definition = "dynamic"
-groups = []
-image_map = []
-
-[status.session]
-completion_time = "2024-08-31T21:09:19"
-job = "cfs-b840455c-e919-4656-b64a-44d433f082dc"
-start_time = "2024-08-31T21:08:47"
-status = "complete"
-succeeded = "True"
-```
-
-#### Track the status of Ansible playbooks
-
-Make sure that all the CFS play books have completed successfully.
-Look for the message "All playbooks completed successfully" in the CFS Ansible container log.
-
-See [Troubleshooting](Track_the_Status_of_a_Session.md) for more information.
-
-1. Get name of CFS pod.
-
-    ```bash
-    CFSPOD=$(kubectl get pods --no-headers -o custom-columns=":metadata.name" -n services -l cfsession=${SESSION}) ; echo $CFSPOD
-    ```
-
-    Example output:
-
-    ```text
-    cfs-b840455c-e919-4656-b64a-44d433f082dc-5d4jp
-    ```
-
-1. View the last line of Ansible container log.
-
-    ```bash
-    kubectl logs -n services "${CFSPOD}" -c ansible --tail=1
-    ```
-
-    Example of output if CFS was successful:
-
-    ```text
-    All playbooks completed successfully
-    ```
-
-### Post-personalization verification
+# Post-personalization Verification
 
 (`ncn-w#`) Run these checks on each worker node, in order to verify that it was configured correctly.
```

Comment:
If they run this on every worker node, won't they see issues for the workers that are not enabled as iSCSI targets? Or are some of these checks that iSCSI is configured (and therefore would apply to all worker nodes) and some of these are checks that the worker is enabled as an iSCSI target (which may not work on all worker nodes)?